components/common/header/authNav.js

```diff
@@ -1,7 +1,7 @@
 import { useState } from 'react'
 import { useRouter } from 'next/router'
 
-import { logout } from '../../client'
+import { logout } from '../../../client'
```

Comment:
instead of this, do 

import { useAuth} from '../../../utils/useAuth'

and then do
`auth = useAuth()` 

and call logout with
`await auth.logout()`

Reply:
Done! I was getting an error `Redundant use of `await` on a return value.` but it works without it! Testing login in and logging out and it works fine now.

Reply:
perf